managed/services/telemetry/telemetry.go

```diff
@@ -303,3 +303,12 @@ func (s *Service) Format(report *reporter.ReportRequest) string {
 
 	return builder.String()
 }
+
+// GetSummaries returns the list of gathered telemetry
+func (s *Service) GetSummaries() []string {
```

Comment:
if telemetry is disabled should we return an empty array?

Reply:
Not sure about this. 
This method is used to get summaries of datapoints to display them on UI, in the Advanced Settings page.

Reply:
the UI just uses this to show a tooltip on what telemetry would be collected if enabled, so we need to be able to show it all the time